abr_control/controllers/floating_task.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+
+from . import controller
+
+
+class FloatingTask(controller.Controller):
```

Comment:
rename `floating` to `floating_joint` and made a `floating_task` version as well

Reply:
ooh yeah let's bring these in